pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
   'circus~=0.18.0',
   'click-spinner~=0.1.8',
   'click~=8.1',
-  'disk-objectstore~=1.1',
+  'disk-objectstore~=1.1.1',
```

Comment:
Thanks @unkcpz for the review

I understand that `disk-objectstore~=1.1`  would automatically install the latest patch release. 
However, I think it's better to clearly specify the minimum compatible version, as `1.1.0` is not compatible.

This is important because if a user already has `disk-objectstore v1.1.0` installed and loses internet access during the installation of `aiida-core`, the installation will proceed with `disk-objectstore v1.1.0` without requiring it to update to `1.1.1`. 
In that case, we have a wrong installation, with no warning or whatsoever that leads to a bunch of errors emerging during the runtime.

Reply:
> as 1.1.0 is not compatible.

What you mean by "not compatible"? If it is a release that have problem then maybe we need to bring it down from pypi. If `1.1.1` is just adding new feature then having `v1.1.0` should be fine I guess?

Reply:
Ah, I see. The implementation here is rely on the change with the release `1.1.1`. 
Then if we follow the semver ("MINOR version when you add functionality in a backward compatible manner"), that one should be released as a minor version bump.

Reply:
I just did a read on https://github.com/aiidateam/AEP/blob/master/002_dependency_management/readme.md again.
If we gonna to follow it, then we probably need to release `v1.2` and using `~=1.2` in `aiida-core`.

Reply:
Ok, I understand that should have been released as `1.2`.
But now, given the situation I don't really think we need to re-release `disk-objectstore` just to follow semantic versioning thing.

Time and effort it requires, does payoff for the fruit.

Reply:
I can understand the frustration, but that is not in the scope of this PR to solve other peoples mistakes, but in this case we did a wrong release.